core/src/main/java/com/linecorp/armeria/server/ServiceNaming.java

```diff
@@ -72,6 +73,16 @@ static ServiceNaming simpleTypeName() {
         };
     }
 
+    /**
+     * Returns the {@link ServiceNaming} that returns the shortened service name from the full name of an RPC
+     * stub class or the innermost class from the given service. It follows Logback's abbreviation algorithm.
```

Comment:
- Add an example for a shortening name?
- Add a link for `Logback's abbreviation algorithm.`?